crypto/txscript/src/opcodes/mod.rs

```diff
@@ -566,54 +585,38 @@ opcode_list! {
 
     // Numeric related opcodes.
     opcode Op1Add<0x8b, 1>(self, vm) {
-        let [value]: [i64; 1] = vm.dstack.pop_items()?;
-        vm.dstack.push_item(value + 1);
-        Ok(())
+        numeric_op!(vm, [value], 1, value.checked_add(1).ok_or_else(|| TxScriptError::NumberTooBig("todo".to_string()))?)
```

Comment:
You want me to rename the error? it will break any code that depends on that.
If i add new variant, it won't be clear what to chose NumberTooBig or Overflow

Reply:
commit 3259cc61aa3082977d8b9a5094efd4895c262008